code/LINDA/LINDA_turf_tile.dm

```diff
@@ -352,8 +352,9 @@ atom/movable/proc/experience_pressure_difference(pressure_difference, direction)
 	for(var/turf/simulated/T in turf_list)
 		T.excited = 0
 		T.recently_active = 0
+		T.excited_group = null
```

Comment:
I or you can talk to Aran about this, but he definitely set it to be this intentionally: https://github.com/tgstation/-tg-station/commit/ea5ec6aec2d54e662f69bf3b332c98b5a89102f2#diff-66ee58580795f0aa9d72744f56e1bd6dR357